src/main/java/at/ac/tuwien/kr/alpha/Main.java

```diff
@@ -81,6 +83,9 @@ public static void main(String[] args) {
 		filterOption.setValueSeparator(',');
 		options.addOption(filterOption);
 
+		Option deterministicOption = new Option("d", OPT_DETERMINISTIC, true, "disable randomness");
```

Comment:
I think the third parameter should be `false` to indicate that the option takes no argument?